public/components/datasources/components/manage/accelerations/acceleration_table.tsx

```diff
@@ -68,6 +69,7 @@ export const AccelerationTable = ({
     actionType: null,
     selectedItem: null,
   });
+  const { loadStatus, startLoading, stopLoading } = useLoadTableColumnsToCache();
```

Comment:
should this be declared in dataconnections page? What happens when we close flyout and switch the tab to associated objects

Reply:
Associated objects tab also initializes the hook and passes it into the `Create Acceleration` flyout. The main issue was that when the flyout unmounts, we lose the reference to `stopLoading`, so the polling in `startLoading` won't stop. Lifting it out to the nearest parent component ensures that `stopLoading` still exists after the flyout unmounts and we can safely stop the loading.